README.md

```diff
@@ -127,6 +141,17 @@ publish.standard();
         </tr>
         <tr>
             <td>
+                <code>checkRemoteCmd</code>
```

Comment:
This material doesn't appear in this diff (yet) but this is the closest line:

i) There's a typo where "--standard" appears twice, presumably the first instance should be "--version"
ii) The heading named "Node" is pretty confusing and it's unclear what it refers to. There should be some text here to explain what is being documented
iii) There then seems to be some kind of missing heading since the docs then continue with "parameters" again which is a duplicate.
iv) The JSON-encoded form for accepting "options" will be pretty hard for users to use. We should instead accept standard UNIX-style key-value pairs which will be applied to the options individually - for example,

fluid-publish remoteName="origin" --standard

etc.